internal/ciphersuite/tls_ecdhe_ecdsa_with_aes_128_gcm_sha256.go

```diff
@@ -47,14 +47,7 @@ func (c *TLSEcdheEcdsaWithAes128GcmSha256) IsInitialized() bool {
 	return c.gcm.Load() != nil
 }
 
-// Init initializes the internal Cipher with keying material
-func (c *TLSEcdheEcdsaWithAes128GcmSha256) Init(masterSecret, clientRandom, serverRandom []byte, isClient bool) error {
-	const (
-		prfMacLen = 0
-		prfKeyLen = 16
-		prfIvLen  = 4
-	)
-
+func (c *TLSEcdheEcdsaWithAes128GcmSha256) init(masterSecret, clientRandom, serverRandom []byte, isClient bool, prfMacLen, prfKeyLen, prfIvLen int) error {
```

Comment:
Why are we splitting out part of the Init function in an internal init function? Based on what I can see the implementation is identical and I don't see what we're gaining by doing this?

Reply:
Ah nvm, got it, TLSEcdheEcdsaWithAes256GcmSha384 is using it.